plugin-ci/orb.yml

```diff
@@ -104,11 +104,22 @@ jobs:
       - checkout
       - *restore_cache
       - run: make dist
+      - run:
+          name: "Generating release notes"
+          command: |
+            echo "### Supported Mattermost Server Versions: $(cat plugin.json | jq .min_server_version -r)+"  >> release-notes.md
```

Comment:
What does this output if `min_server_version` is not set? Should this case be handled, e.g. with an error?

Reply:
When the key is absent it outputs `-### Supported Mattermost Server Versions: null+` whereas when the value of the key is empty this is `-### Supported Mattermost Server Versions: +` the output. While creating release notes checking `min_server_version` might not be the right idea because we release once in a while. Can we keep this check in some other job so that the commit which causes this breaks the CI build?

Reply:
Do you mean another CI job? It seams to "little" to me to check in a CI build.

Reply:
No. I meant to add a step in the existing orb which would fail the CI build when `min_server_version` is absent.